runhttpd.sh

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/bash
 
-IP=${IP:-"172.22.0.1"}
+PROVISIONING_INTERFACE=${PROVISIONING_INTERFACE:-"provisioning"}
 HTTP_PORT=${HTTP_PORT:-"80"}
+HTTP_IP=`ip -4 address show dev $PROVISIONING_INTERFACE | grep -oP '(?<=inet\s)\d+(\.\d+){3}' | head -n 1`
```

Comment:
please use $() instead of backquotes to wrap the command
let's wrap $PROVISIONING_INTERFACE in double-quotes "